modules/project-comments/client/views/period-list.html

```diff
@@ -58,76 +58,81 @@ <h2>Public Comment Period <span class="break">is Now Open</span></h2>
-                  title="Delete Comment Period"
-                  ng-click="$event.stopPropagation();"
-                  x-confirm-dialog
-                  x-title-text="'Delete Comment Period'"
-                  x-ok-text="'Yes'"
-                  x-cancel-text="'No'"
-                  x-confirm-text="'Are you sure you would like to permanently delete this comment period and all its content?'"
-                  x-on-ok="s.deletePeriod"
-                  x-ok-args="p">
-                  <span class="glyphicon glyphicon-trash"></span>
-                </button>
-              </li>
-            </div>
+
+            <script type="text/ng-template" id="pcp-list-menu.html">
+              <ul>
+                <li ng-if="project.userCan.createCommentPeriod">
+                  <a ui-sref="p.commentperiod.edit({periodId:p._id})" ng-click="$event.stopPropagation();">
+                    <span class="glyphicon glyphicon-pencil"></span>Edit Comment Period</a>
+                  </li>
+                <li ng-if="!p.isPublished && project.userCan.createCommentPeriod">
+                  <a title="Publish Comment"
+                    ng-click="$event.stopPropagation();"
+                    x-pcpvalidation-dialog
+                    x-title-text="'Publish Comment Period'"
+                    x-ok-text="'Yes'"
+                    x-cancel-text="'No'"
+                    x-confirm-text="'Are you sure you want to publish this comment period?'"
+                    x-on-ok="s.publishCommentPeriod"
+                    x-period="p">
+                    <span class="glyphicon glyphicon-ok-circle"></span>Publish
+                </a>
+                </li>
+                <li ng-if="p.isPublished && project.userCan.createCommentPeriod">
+                  <a title="Unpublish Comment"
+                    ng-click="$event.stopPropagation();"
+                    ng-disabled="p.openForComment"
+                    x-confirm-dialog
+                    x-title-text="'Unpublish Comment Period'"
+                    x-ok-text="'Yes'"
+                    x-cancel-text="'No'"
+                    x-confirm-text="'Are you sure you want to unpublish this comment period?'"
+                    x-on-ok="s.unpublishCommentPeriod"
+                    x-ok-args="p">
+                    <span class="glyphicon glyphicon-ban-circle"></span>Unpublish
+                  </a>
+                </li>
+                <li ng-if="project.userCan.createCommentPeriod && p.openForComment">
+                  <a title="Delete Comment Period"
+                    ng-click="$event.stopPropagation();"
+                    x-confirm-dialog
+                    x-title-text="'Delete Comment Period'"
+                    x-ok-text="'Yes'"
+                    x-cancel-text="'No'"
+                    x-confirm-text="'This comment period is currently open for public comment, are you sure you would like to permanently delete this comment period and all its content?'"
```

Comment:
The two sentences above this use "Are you sure you want to..."
This one, and the one below, use "are you sure you would like to..."

Not particularly important, but should these both use the same thing: "want to"?

Reply:
We can address this in another ticket to improve the technical writing. I'll review with PO and create another task if wanted.